stetho/src/main/java/com/facebook/stetho/inspector/elements/Descriptor.java

```diff
@@ -55,6 +57,41 @@ public final void postAndWait(Runnable r) {
     getHost().postAndWait(r);
   }
 
+  /**
+   * Parses the text argument text from DOM.setAttributeAsText()
+   * Text will be in the format "attribute1=\"Value 1\" attribute2=\"Value2\""
+   * @param text the text argument to be parsed
+   * @return a map of attributes to their respective values to be set.
+   */
+  protected Map<String, String> parseSetAttributesAsTextArg(String text) {
+    char[] chars = text.toCharArray();
```

Comment:
making a copy of a String to a char-array for the purpose of looping over it is kind of a bad idea, performance-wise. Should use a regular for loop like so,
```
for (int i = 0; i < text.length(); ++i) {
  final char c = text.charAt(i);
  ...
}
```